src/routes/transactions/mappers/common/swap-order.mapper.ts

```diff
@@ -29,6 +36,13 @@ export class SwapOrderMapper {
       orderUid,
     });
 
+    if (
+      this.restrictApps &&
+      !this.allowedApps.has(order.fullAppData['appCode'])
```

Comment:
I think it would be good to add a comment with a link to the CoW docs here. What do you think?

Reply:
Maybe in the Config itself what do you think?